docs/control_center_docs/servo_module_getting_started.rst

```diff
@@ -0,0 +1,111 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _servo_module_start_guide:
+
+####################################################################
+Getting Started with Vertiq's Servo Firmware with IQ Control Center
+####################################################################
+
+Before completing the following getting started guide, please ensure that you have read and completed our :ref:`IQ Control Center 
+guide <control_center_start_guide>`. It walks you through Control Center installation, module configuration, and basic testing options available 
+to all Vertiq modules and firmware types. The following document is meant to provide additional configurations and testing options available only on Vertiq's 
+servo firmware.
+
+.. note::
+    The following images are captured using IQ Control Center version 1.5.2 as connected to a Vertiq 40-06 Gen 1
```

Comment:
I'm not sure that such a thing as the 40-06 Gen 1 really exists. The Gen 2 stuff is for all the modular things, and that's the only kind of 40 we have. There are betas vs. production modules. Probably just say Gen 2 here